manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java

```diff
@@ -78,6 +85,17 @@ public class BridgesServiceImpl implements BridgesService {
     @Inject
     ProcessingErrorService processingErrorService;
 
+    @PostConstruct
+    void init() {
+        if (!Objects.isNull(b64TlsCertificate) && !b64TlsCertificate.isEmpty()) {
+            LOGGER.info("Decoding base64 tls certificate and key");
+            tlsCertificate = new String(Base64.getDecoder().decode(b64TlsCertificate));
+            tlsKey = new String(Base64.getDecoder().decode(b64TlsKey));
+        } else {
```

Comment:
@r00ta For some environments this should be a failure condition e.g in `prod` the Fleet Manager should fail to start if the SSL certs are not configured. If it's a quick fix can we do it here, otherwise let's unblock ourselves in `dev` and do a follow up PR. 

cheers,

Rob